scalr/resource_scalr_variable.go

```diff
@@ -231,6 +269,22 @@ func resourceScalrVariableRead(ctx context.Context, d *schema.ResourceData, meta
 	_ = d.Set("sensitive", variable.Sensitive)
 	_ = d.Set("description", variable.Description)
 	_ = d.Set("final", variable.Final)
+	_ = d.Set("updated_by_email", variable.UpdatedByEmail)
+
+	if variable.UpdatedAt != nil {
+		_ = d.Set("updated_at", variable.UpdatedAt.Format(time.RFC3339))
```

Comment:
What is the reason to convert iso8601 to rfc3339?

Reply:
updated_at is a string in the resource schema, so the time.Time must be converted. in other case an error is raised

Reply:
@igor-melnyk-scalr how is it different from this example? https://github.com/Scalr/terraform-provider-scalr/blob/a92bb6bda8a9f532a359ea1f7de6b6a783dbc087/scalr/resource_scalr_webhook.go#L260
Just want to make sure we work consistently with dates.

Reply:
I saw this case, but I got error when I assign value without formatting